man/adoc/bpftrace.adoc

```diff
@@ -232,20 +233,21 @@ For more details see the <<Verbose Output>> section.
 Programs saved as files are often called scripts and can be executed by specifying their file name.
 We use a `.bt` file extension, short for bpftrace, but the extension is not required.
 
-For example, listing the sleepers.bt file using `cat -n`:
+For example, listing the sleepers.bt file using `cat`:
 
 ----
-# cat -n sleepers.bt
-1 tracepoint:syscalls:sys_enter_nanosleep
-2 {
-3   printf("%s is sleeping.\n", comm);
-4 }
+$ cat sleepers.bt
+
+tracepoint:syscalls:sys_enter_nanosleep {
+  printf("%s is sleeping.\n", comm);
+}
 ----
 
 And then calling it:
 
 ----
-# bpftrace sleepers.bt
+$ bpftrace sleepers.bt
```

Comment:
```suggestion
$ sudo bpftrace sleepers.bt
```
I don't know if there's any kind of standard, but in my experience `$` is usually the prompt for a non root user (root has `#`)

Reply:
Ha. I did a brief search and couldn't find any docs on when to use which but I'll follow your lead.

Reply:
Changed these back to `#` to be consistent with the rest of the adoc.